s3_file_uploads/aws.py

```diff
@@ -15,14 +16,15 @@ def __init__(self, id: str) -> None:
         self.id = str(id)  # coerce just in case someone passes in a UUID()
         self.s3 = boto3.client('s3')
 
-    def get_upload_form(self, expiry=UPLOAD_ENDPOINT_EXPIRES) -> dict:
+    def get_upload_form(self, expiry=UPLOAD_ENDPOINT_EXPIRES, acl_type=None) -> dict:
         # NOTE: Used because it's the only way to enforce upload
         #       size on S3.
         return self.s3.generate_presigned_post(
             settings.AWS_BUCKET_NAME,
             self.id,
             Fields={},
             Conditions=[
+                acl_type or {ACL_KEY: PRIVATE},
```

Comment:
This setup would allow users to pass something which isn't an ACL into the `acl_type` parameter. Maybe simply forcing `{ACL_KEY: acl_type}` here, and do the default setting in the  method itself?

Reply:
I'm not saying that a different implication isn't better, but the serialiser which supplies this data always filters for key `acl`

Reply:
Agreed as discussed and updated